third_party/silabs/silabs_board.gni

```diff
@@ -41,6 +41,9 @@ declare_args() {
   #Disable MQTT by default
   enable_dic = false
 
+  #Disbale DIC nvm embed cert by default
```

Comment:
Do we need a new arg and define for this? can't `enable_dic` be used?
Is there a use case where DIC is used but not the DIC certs?

Reply:
@Thirsrin an answer to this comment would be appreciated